utils/print_bingo_card_image.py

```diff
@@ -1,85 +1,50 @@
-    draw.text(((center_column_one - px_removal_to_center(song_2)), (center_row_two - spacing_rows)), song_2, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(artist_2)), (center_row_two + spacing_rows)), artist_2, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(song_3)), (center_row_three - spacing_rows)), song_3, fill="black", font=font)
-    draw.text(((center_column_one - px_removal_to_center(artist_3)), (center_row_three + spacing_rows)), artist_3, fill="black", font=font)
-
-    draw.text(((center_column_two - px_removal_to_center(song_4)), center_row_one - spacing_rows), song_4, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_4)), center_row_one + spacing_rows), artist_4, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(song_5)), (center_row_two - spacing_rows)), song_5, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_5)), (center_row_two + spacing_rows)), artist_5, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(song_6)), (center_row_three - spacing_rows)), song_6, fill="black", font=font)
-    draw.text(((center_column_two - px_removal_to_center(artist_6)), (center_row_three + spacing_rows)), artist_6, fill="black", font=font)
-
-    draw.text(((center_column_three - px_removal_to_center(song_7)), center_row_one - spacing_rows), song_7, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_7)), center_row_one + spacing_rows), artist_7, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(song_8)), (center_row_two - spacing_rows)), song_8, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_8)), (center_row_two + spacing_rows)), artist_8, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(song_9)), (center_row_three - spacing_rows)), song_9, fill="black", font=font)
-    draw.text(((center_column_three - px_removal_to_center(artist_9)), (center_row_three + spacing_rows)), artist_9, fill="black", font=font)
-
-    image_rotated = image.rotate(90)
+import textwrap
+
+def make_bingo_card(image_path, df, card_size=9):
+    image_path = r'n:\My Documents\Julien\iQuizzen\music_bingo\scraps\template.jpg'
+    img = Image.open(image_path)
+    draw = ImageDraw.Draw(img)
+    font = ImageFont.truetype("cour.ttf", 20)
+
+    line_width,line_height = draw.textsize('-', font)
+    x_center = [170, 450, 720] # center-coordinate (x-axis) of each columnn
+    y_center = [290, 525, 751] # center-coordinate (y-axis) of each row
+    char_max = 14 # max nr of characters
+
+    num_rows_cols = int(len(df) ** 0.5)
+    for row in range(num_rows_cols):
+        for col in range(num_rows_cols):
+            ind = row*3+col
```

Comment:
What does ind mean?

Reply:
ind = index. Er wordt over de 3 rijen en 3 kolommen geloopt (en/of over de 9 titels+artiesten in de dataframe). 
Ik zal _ind_ vervangen door _index_